src/videojs-hls.js

```diff
@@ -995,7 +1019,7 @@ videojs.HlsHandler.prototype.loadSegment = function(segmentInfo) {
       return;
     }
 
-    self.lastSegmentLoaded_ = segmentInfo.uri;
+    self.lastSegmentLoaded_ = segment;
```

Comment:
Are you referring to the `this.lastSegmentLoaded_ === this.playlistUriToUrl(segment.uri)` check on line 857? I had updated that comparison to look at the entire segment, but per your comment [here](https://github.com/videojs/videojs-contrib-hls/pull/454/files#r46886998) it sounds like that won't work. Is there some other way we can uniquely identify segments there? The quick and dirty option would be checking both URI and byterange as part of that if statement, but I'm not sure if that's up to your standards.

I've been away for a bit but have some bandwidth now to fix this up. Would you be okay with a quick fix like the one I suggest? Would you prefer a different type of identifier? Or is this not worth tackling right now?

I think this is the only other place in the file where we use `lastSegemntLoaded_`. If we can take care of this, it should be ready to go. Let me know what I can do to help.